Dockerfile.tests

```diff
@@ -0,0 +1,16 @@
+FROM golang:1
+
+ENV PROJECT=concepts-rw-neo4j
```

Comment:
`concepts-rw-neo4j` ?

Reply:
Well stack overflow stared blocking my copy-paste. I needed to get the code from somewhere ...